keras_contrib/applications/resnet.py

```diff
@@ -490,10 +492,16 @@ def ResNet(input_shape=None, classes=10, block='bottleneck', residual_unit='v2',
 
     if time_distributed:
         if len(input_shape) != 4:
-            raise Exception("Input shape should be a tuple (nb_time_steps, nb_rows, nb_cols, nb_channels)")
+            if K.image_data_format() == 'channels_last':
```

Comment:
Sorry for the late observation, this check for input_shape dimensions needs to be before "# Determine proper input shape".